pages/pipelines/templates.md

```diff
@@ -0,0 +1,62 @@
+# Pipeline templates
+
+> 📘 Enterprise feature
+> Pipeline templates is only available on an [Enterprise](https://buildkite.com/pricing) plan.
```

Comment:
Nitpick: This works when you think about "Pipeline templates" as a feature name, but I think it reads more smoothly with "are." Happy for you to decide which way to go.

```suggestion
> Pipeline templates are only available on an [Enterprise](https://buildkite.com/pricing) plan.
```